Formula/cayley.rb

```diff
@@ -1,56 +1,64 @@
 class Cayley < Formula
-  desc "Graph database inspired by Freebase and Knowledge Graph"
-  homepage "https://github.com/cayleygraph/cayley"
+  desc "An open-source graph database"
+  homepage "https://cayley.io/"
   url "https://github.com/cayleygraph/cayley.git",
     :tag      => "v0.7.7",
     :revision => "dcf764fef381f19ee49fad186b4e00024709f148"
+  sha256 "37ef9043cb20aeffa93d57f360fdee3e0c9d1b2fa8d404a00f3bff1b58937206"
```

Comment:
If you want to change sha256 without changing the version, you may want to use `revision`.
See https://docs.brew.sh/Formula-Cookbook#formulae-revisions.

Reply:
In addition to that, why did the hash change? Is upstream aware of this and was it intentional?

Reply:
Sorry, I misunderstood the situation.
The sha256 is *not* changed, but added by mistake.
The source is downloaded via git, so the formula doesn't need the sha256.

---

@iddan Please remove the sha256, since this is not needed.

Reply:
Okay, will do!